test/integration/user_show_test.rb

```diff
@@ -0,0 +1,19 @@
+require "test_helper"
+class UserVisitsPlaceSpec < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+  require 'vcr'
+  require 'webmock'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.